test/test.js

```diff
@@ -256,4 +256,154 @@ describe('ConfigGenerator', function () {
             });
         });
     });
+
+    it('should rewrite "define" calls if "namespace" option is present', function (done) {
```

Comment:
This title is wrong since it's testing exactly the opposite: 

```javascript
it('should NOT rewrite "define" calls if "skipFileOverride" is true, even if "namespace" option is present', {...});
```